GenderPayGap.WebUI/Search/CachedObjects/SearchReadyValue.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace GenderPayGap.WebUI.Search.CachedObjects
+{
+    public class SearchReadyValue
+    {
+
+        public string OriginalValue { get; }
+
+        public List<string> LowercaseWords { get; }
+
+        public string Acronym { get; }
+
+
+        public SearchReadyValue(string originalValue)
+        {
+            OriginalValue = originalValue;
+            LowercaseWords = SplitValueIntoWords(originalValue);
+            Acronym = MakeAcronymFromWords(LowercaseWords);
+        }
+
+        private static List<string> SplitValueIntoWords(string originalValue)
+        {
+            Match match = WordSplittingRegex.Regex.Match(originalValue);

```

Comment:
Will this regex also remove any punctuation (eg. co-op => coop etc)?

Reply:
Good point, I missed that case.
Just about to push a change that will include this.